src/components/Map/USACountyMap.js

```diff
@@ -39,6 +56,25 @@ const USACountyMap = ({ stateClickHandler, setTooltipContent, condensed }) => {
               geographies.map(geo => {
                 const { name } = geo.properties;
                 const stateCode = getStateCode(name);
+
+                if (stateCode === 'MP') {
+                  return (
+                    <Link key={stateCode} to={`/us/${stateCode.toLowerCase()}`}>
+                      <MarianaIslands
+                        x={40}
+                        y={395}
+                        scale={0.8}
```

Comment:
Would it make sense to just hardcode these in the MarianaIslands component?

Reply:
I hesitated because i was thinking of the MarianaIslands component as more standalone that could be placed anywhere, but don't think it adds much value so fine moving it in the component itself